routes/ninja-routes.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+const Ninja = require('../models/ninjas')
+
+module.exports = function(router) {
+  router.get('/api/ninja', (req, res) => {
+    // if(!req.params.id) return res.status(400).send(new Error('Bad Request'))
+    if(!req.query.id) {
```

Comment:
So with the lab, you'll need to use a controller, and in that controller you'll have this logic.